app/models/card.rb

```diff
@@ -11,9 +14,12 @@ def equal_text
       errors.add(:translated_text, "Перевод не может быть равен оригиналу")
     end	
   end
-  
-  protected
-    def add_review_time
-      self.review_date = Date.today + 3.day
-    end
+
+  def set_review_date
+    self.review_date = Date.today	
+  end
+
+  def add_review_date
+    self.update_attribute(:review_date, Date.today + 3.day)
```

Comment:
Redundant `self` detected.<br>Do not use `Date.today` without zone. Use `Time.zone.today` instead.